DependencyInjection/Configuration.php

```diff
@@ -20,6 +20,23 @@
 class Configuration implements ConfigurationInterface
 {
     /**
+     * Whether to use the debug mode.
+     *
+     * @see https://github.com/doctrine/DoctrineBundle/blob/master/DependencyInjection/Configuration.php#L31-L41
```

Comment:
If you want to have a reference to a line you should probably do it to a tag instead of master: 
https://github.com/doctrine/DoctrineBundle/blob/v1.5.2/DependencyInjection/Configuration.php#L31-L41